### PR TITLE
Adjust post-review weight calculation for negative reviews

### LINKED ISSUE #5473: Prioritize post-review queue with risk factors evaluation
For post-review, we need to sort the list of auto-approved add-ons in reviewer tools according to a set of factors representing the "risk" to end-users.

The set of factors is defined here: https://docs.google.com/spreadsheets/d/1CZEECHHqEmK87fNiWNslAsFQBL7zxYF-c63-zZVZuqw/edit#gid=0

It would be set at (auto-)approval time, stored in a new field in `AutoApprovalSummary` and then used to sort the list in reviewer tools.

### PR DESCRIPTION
**Pull request by diox**
It should be 1% of total number of reviews with score less than 3, multiplied by 2.

(I initially thought the 1% was a typo, but it was confirmed as correct)

(Re-) Fix #5473

### CODE REVIEW

File: src/olympia/editors/models.py [hunk 1]
```diff
@@ -732,13 +732,13 @@ def calculate_weight(self):
...
-            # weight, up to a maximum of 100.
-            'negative_reviews': min(
+            # 1% of the total of "recent" reviews with a score of 3 or less
+            # adds 2 to the weight, up to a maximum of 100.
+            'negative_reviews': min(int(
                 Review.objects
                 .filter(addon=addon)
                 .filter(rating__lte=3, created__gte=one_year_ago)
-                .count() * 2, 100),
+                .count() / 100. * 2), 100),
```
Comment (EnTeQuAk) [inline src/olympia/editors/models.py]: If you use float-notation, I'd prefer using float notation for the whole expression, e.g `.count() / 100.0 * 2.0` which makes it a bit easier to discover.

File: src/olympia/editors/tests/test_models.py [hunk 2]
```diff
@@ -767,21 +767,27 @@ def test_calculate_weight_negative_reviews(self):
...
             rating=2)
-        summary = AutoApprovalSummary(version=self.version)
         weight_info = summary.calculate_weight()
-        assert summary.weight == 4
-        assert weight_info['negative_reviews'] == 4
+        assert summary.weight == 1
+        assert weight_info['negative_reviews'] == 1
 
-        # Create fifty more to make sure it's capped at 100.
+        # Create 5000 more (sorry!) to make sure it's capped at 100.
```
Comment (EnTeQuAk) [inline src/olympia/editors/tests/test_models.py]: shouldn't 200 or 300 be enough already? I don't mind us creating that many more objects in tests if we need them but 5000 seems overkill :)

Comment (diox) [inline src/olympia/editors/tests/test_models.py]: No, since we want to test the cap (100), so 200 or 300 would only result in a score of 4 - 6...

Comment (EnTeQuAk) [inline src/olympia/editors/tests/test_models.py]: I see, fair point. Let's stay with 5000 then.

**Pull request merged**